book/src/validator-start.md

```diff
@@ -49,7 +49,7 @@ commands:
 $ solana-keygen new -o ~/validator-vote-keypair.json
 $ VOTE_PUBKEY=$(solana-keygen pubkey ~/validator-vote-keypair.json)
 $ IDENTITY_PUBKEY=$(solana-keygen pubkey ~/validator-keypair.json)
-$ solana-wallet create-vote-account "$VOTE_PUBKEY" "$IDENTITY_PUBKEY" 1
+$ solana-wallet --keypair ~/validator-keypair.json create-vote-account "$VOTE_PUBKEY" "$IDENTITY_PUBKEY" 1
```

Comment:
Can we just remove VOTE_PUBKEY and IDENTITY_PUBKEY entirely now?

Reply:
Are you asking if we can remove the variable assignations? Or remove the args entirely?
If the former, yes, we can just rewrite the solana-wallet command.

Reply:
The former :)

Reply:
Done